apps/server/src/modules/fwu-learning-contents/uc/fwu-learning-contents.uc.spec.ts

```diff
@@ -0,0 +1,213 @@
+import { S3Client } from '@aws-sdk/client-s3';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { InternalServerErrorException, NotFoundException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { Logger } from '@src/core/logger';
+import { Readable } from 'stream';
+import { S3Config } from '../interface/config';
+import { FwuLearningContentsUc } from './fwu-learning-contents.uc';
+
+const createParameter = () => {
```

Comment:
I would prefer this to be defined within the scope of at least the outermost describe

Reply:
Put it in the outermost 'describe'. It was out of the 'describe' because files-storage did it as well.